backbone/src/main/res/values/strings.xml

```diff
@@ -175,4 +175,9 @@
     <string name="rsb_consent_role">Participant</string>
     <string name="rsb_consent_doc_line_date_format">"MM-dd-yyyy"</string>
 
+    <!-- Edit -->
+    <string name="rsb_consent_review_save">Save</string>
```

Comment:
is this (only added here in english) intended? the `rsb_consent_review_save` is here alone and not in the other files.